lib/Mojolicious.pm

```diff
@@ -260,6 +264,19 @@ built-in ones (except for L<Mojo::Server::CGI>).
 Useful for reconfiguring application servers dynamically or collecting server diagnostics information. (Passed the
 server and application objects)
 
+=head2 before_worker_start
+
+Emitted right before the application starts, regardless of whether it
+is invoked via C<morbo>, via the command line in plain daemon mode or
+through pre-forking via C<hypnotoad>.
```

Comment:
There is no need to list individual servers, the `before_server_start` hook doesn't do that either. Just follow its example. This hook is almost the same with different timing.

Reply:
Also needs reformatting.

Reply:
Reformatting?  how so?

(not to be picky, but it's not just the *timing*, it's also the *place* where it is run)

Reply:
Anyway, this paragraph is now gone.